### PR TITLE
GPT-4V changed to GPT-4o

### PR DESCRIPTION
**Pull request by MW-OAI**
Change model to GPT-4o
Adding image resizing the manage token limit

## Summary

Model change from GPT-4V to GPT-4o

## Motivation

Why are these changes necessary? How do they improve the cookbook?

---

## For new content

When contributing new content, read through our [contribution guidelines](https://github.com/openai/openai-cookbook/blob/main/CONTRIBUTING.md), and mark the following action items as completed:

- [ ] I have added a new entry in [registry.yaml](https://github.com/openai/openai-cookbook/blob/main/registry.yaml) (and, optionally, in [authors.yaml](https://github.com/openai/openai-cookbook/blob/main/authors.yaml)) so that my content renders on the cookbook website.
- [ ] I have conducted a self-review of my content based on the [contribution guidelines](https://github.com/openai/openai-cookbook/blob/main/CONTRIBUTING.md#rubric):
  - [ ] Relevance: This content is related to building with OpenAI technologies and is useful to others.
  - [ ] Uniqueness: I have searched for related examples in the OpenAI Cookbook, and verified that my content offers new insights or unique information compared to existing documentation.
  - [ ] Spelling and Grammar: I have checked for spelling or grammatical mistakes.
  - [ ] Clarity: I have done a final read-through and verified that my submission is well-organized and easy to understand.
  - [ ] Correctness: The information I include is correct and all of my code executes successfully.
  - [ ] Completeness: I have explained everything fully, including all necessary references and citations.

We will rate each of these areas on a scale from 1 to 4, and will only accept contributions that score 3 or higher on all areas. Refer to our [contribution guidelines](https://github.com/openai/openai-cookbook/blob/main/CONTRIBUTING.md) for more details.

### CODE REVIEW

File: examples/Parse_PDF_docs_for_RAG.ipynb
Comment (teomusatoiu) [inline examples/Parse_PDF_docs_for_RAG.ipynb:1]: Thanks for this, here are a few comments: 
- Under File Processing the variable page_text doesn't seem to be used, you can remove 
- Make sure you run all cells in your notebook you have some that you haven't run overall (in testing with an example, parsing all documents, etc. ) - I would suggest taking the notebook and re-running it cell by cell to make sure the outputs are correct.

**Review by teomusatoiu — CHANGES_REQUESTED** [review]
Thanks for this! Just a few comments!

File: examples/Parse_PDF_docs_for_RAG.ipynb
Comment (MW-OAI) [reply examples/Parse_PDF_docs_for_RAG.ipynb:1]: Thanks, made requested changes

**Review by teomusatoiu — APPROVED** [review]
Nice job! approved!

**Pull request merged**